flake.nix

```diff
@@ -27,6 +27,13 @@
       flake = false;
     };
 
+
+    # for testing the ogmios parser
+    ogmios-fixtures = {
+      url = "github:CardanoSolutions/ogmios?ref=master";
```

Comment:
We also want the fixtures to match the version of Ogmios we're using, no? Our fork also has these fixtures

Reply:
I kept it separate in case we did want to update our fixtures without having to take new ogmios changes. I also wasn't sure if the flake input ogmios can be treated as a directory, whereas this input is explicitly not a flake. That being said if we are going to combine them, we can move this work into our ogmios fork